docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,266 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fast Fourier Transform (FFT)\n",
```

Comment:
we need to be careful with FFT and DFT, somehow we use it here in the same way, we need to specify that FFT (Fast Fourir Transform) is the the algorithm behind the DFT.